test/pingtimes.ts

```diff
@@ -127,3 +127,35 @@ describe("Pingtimes", function() {
     }
   });
 });
+
+describe("PingtimesReproducesOriginalImplementation", function() {
```

Comment:
Great, that looks comprehensive.

Could you fold the tests back into the top level describe? The pattern here is the [mocha BDD ](https://mochajs.org/#bdd) one:
```
describe("a subsystem"
  describe/context("optional further breakdown"
      it("should do something" ..
```
These backwards compatibility tests would be better to live within the existing `describe("Pingtimes"` function, perhaps along these lines:
```
describe("backwards compatibility with original TagTime", ...
  context("with the default seed"
     beforeEach("reset the seed", ...)
      it("should have the same first ping" ..)
      it("should have the same ping in 2018" ..))
  context("with a different seed"
      it("should have the same first ping" ..)
      it("should generate the same ping in 2018" ..)
```

The goal is that the output of `npm run test:renderer` feels natural and intuitive to read.